docs/how-to/HA.md

```diff
@@ -1,44 +1,87 @@
-# Set up PMM in HA mode
+# Set up PMM in High Availability (HA) mode
 
 !!! caution alert alert-warning "Important"
-    This feature has been added in PMM 2.41.0 and is currently in [Technical Preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview). Early adopters are advised to use this feature for testing purposes only as it is subject to change.
+    This feature has been added in Percona Monitoring and Management (PMM) 2.41.0 and is currently in [Technical Preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview). Early adopters are advised to use this feature for testing purposes only as it is subject to change.
```

Comment:
since we combined multiple options in 1 page this statement isn't fully true. it's true only for option 4.
option 1 was added in 2.33.0
option 2 existed since 2.28.0
option 3 wasn't implemented yet.